docs/configuration-options.md

```diff
@@ -31,11 +37,11 @@ Be aware that values defined in the build script will override those specified b
 | `iterationTime`                     | Sets the duration for each iteration, both measurement and warm-up.                                                        | Integer                       | @Measurement(..., time: Int, ...)                   |
 | `iterationTimeUnit`                 | Defines the unit for `iterationTime`.                                                                                        | Time unit, see below          | @Measurement(..., timeUnit: BenchmarkTimeUnit, ...) |
 | `outputTimeUnit`                    | Sets the unit for the results display.                                                                                       | Time unit, see below          | @OutputTimeUnit(value: BenchmarkTimeUnit)           |
-| `mode`                              | Selects "thrpt" (Throughput) for measuring the number of function calls per unit time or "avgt" (AverageTime) for measuring the time per function call. | "thrpt", "avgt", "Throughput", "AverageTime"       | @BenchmarkMode                                      |
+| `mode`                              | Selects "thrpt" (Throughput) for measuring the number of function calls per unit time or "avgt" (AverageTime) for measuring the time per function call. | `thrpt`(default), `Throughput`(default), `avgt`, `AverageTime`       | @BenchmarkMode                                      |
```

Comment:
As I have repeated many times: https://github.com/Kotlin/kotlinx-benchmark/pull/117#discussion_r1239269977

Reply:
I will review that pr among others now and in the future before committing doc changes.